app/api/roles.py

```diff
@@ -74,3 +80,45 @@ def before_delete_object(self, obj, kwargs):
                   'methods': {
                       'before_get_object': before_get_object
                   }}
+
+
+@role_misc_routes.route('/change-organiser', methods=['POST'])
```

Comment:
Instead of creating a separate route/blueprint, to follow the JSON-api spec, please use the marshmallow schema like this:
`def after_update_object(self, role, data, view_kwargs):
        """
        Method to update object
        :param role:
        :param data:
        :param view_kwargs:
        :return:
        """
        try:
	        role_entry_organiser_to_delete = UER.query.filter_by(user_id=current_organiser_id,
	                                                             event_id=event_id,
	                                                             role_id=1).one()
	        db.session.delete(role_entry_organiser_to_delete)
	
	    except NoResultFound:
	        return NotFoundError({'source': ''}, 'Organiser Role Entry not found').respond()
	
	    except Exception as e:
	        return ServerError({'source': ''}, e).respond()
	`

Reply:
```python
except Exception as e:
    return ServerError({'source': ''}, e).respond()
```

We should stop doing that ASAP. Bare exceptions should never be caught. They should be thrown

Because of this, a 500 error returns from API and does not get logged or tracked in Sentry